src/runtime-tools/linux/setup.sh

```diff
@@ -159,6 +159,12 @@ if type apt > /dev/null 2> /dev/null; then
     popd
 fi
 
+# enable crash dumps; by default on Ubuntu this is done by apport
+# so we need to disable that
+sudo sysctl -w 'kernel.core_pattern=core' # write dumps to file named 'core'
```

Comment:
this is equivalent to what is done in run.sh right ?
``` bash 
    # use core files, not external crash handler
    echo core | sudo tee /proc/sys/kernel/core_pattern
```

Also, this won't work inside a docker container. 
If you want to keep it in this file you can check for `DOCKER_BUILD` before executing this